distcheck is failing

```
$ make distcheck
[..]
/usr/bin/varnishtest -Dvmoddir=/home/lkarsten/work/varnish-modules/varnish-modules-0.9.1-dev/_inst/lib/varnish/vmods -Dvarnishd=/usr/sbin/varnishd -Dvmod_topbuild=/home/lkarsten/work/varnish-modules/varnish-modules-0.9.1-dev/_build ../../src/tests/saintmode/test02.vtc
[..]
**** v1    0.1 CLI RX| Message from VCC-compiler:\n
**** v1    0.1 CLI RX| Could not load VMOD directors\n
**** v1    0.1 CLI RX| \tFile name: /home/lkarsten/work/varnish-modules/varnish-modules-0.9.1-dev/_inst/lib/varnish/vmods/libvmod_directors.so\n
**** v1    0.1 CLI RX| \tdlerror: /home/lkarsten/work/varnish-modules/varnish-modules-0.9.1-dev/_inst/lib/varnish/vmods/libvmod_directors.so: cannot open shared object file: No such file or directory\n
**** v1    0.1 CLI RX| ('<vcl.inline>' Line 7 Pos 16)\n
**** v1    0.1 CLI RX|         import directors from "/home/lkarsten/work/varnish-modules/varnish-modules-0.9.1-dev/_inst/lib/varnish/vmods/libvmod_directors.so";\n
**** v1    0.1 CLI RX| ---------------#########-------------------------------------------------------------------------------------------------------------------\n
**** v1    0.1 CLI RX| \n
**** v1    0.1 CLI RX| Running VCC-compiler failed, exited with 2\n
**** v1    0.1 CLI RX| VCL compilation failed
---- v1    0.1 FAIL VCL does not compile
*    top   0.1 RESETTING after ../../src/tests/saintmode/test02.vtc
```

Leaving the next distcheck error for @aondio to have a look at.
```
if test -d "varnish-modules-0.9.0"; then find "varnish-modules-0.9.0" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "varnish-modules-0.9.0" || { sleep 5 && rm -rf "varnish-modules-0.9.0"; }; else :; fi
test -d "varnish-modules-0.9.0" || mkdir "varnish-modules-0.9.0"
 (cd src && make  top_distdir=../varnish-modules-0.9.0 distdir=../varnish-modules-0.9.0/src \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory '/home/lkarsten/work/varnish-modules/varnish-modules-0.9.0/_build/sub/src'
make[3]: *** No rule to make target '*vcc', needed by 'distdir'.  Stop.
make[3]: Leaving directory '/home/lkarsten/work/varnish-modules/varnish-modules-0.9.0/_build/sub/src'
Makefile:576: recipe for target 'distdir' failed
make[2]: *** [distdir] Error 1
```

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -66,7 +66,9 @@ $(top_srcdir)/src/tests/*/*.vtc:
 
 check: $(VMOD_TESTS)
 
-EXTRA_DIST = foreign *vcc $(VMOD_TESTS) \
+EXTRA_DIST = foreign \
+	$(top_srcdir)/src/*vcc \
+	$(VMOD_TESTS) \
 	vtree.h
 
 CLEANFILES = $(builddir)/vcc*if.c $(builddir)/vcc*if.h \

